cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "integrationFolder": "test/integration",
+  "videoRecording": false
+}
```

Comment:
I try to avoid more configuration files if possible so I tried this

```
cypress run --config integrationFolder='test/scenarios',videoRecording=false
```

but I still get the error 

```
We looked but did not find a cypress.json file in this folder: /Users/gregor/Projects/octokit/rest
```

Is there a way to work around that?